manifests/virt-handler.yaml.in

```diff
@@ -29,6 +30,8 @@ spec:
         volumeMounts:
         - name: libvirt-runtime
           mountPath: /var/run/libvirt
+        - name: sockets
```

Comment:
I'd prefer not to mix things like sockets and .iso files. Nothing inherently is wrong with that at face value but it might lead to corner cases that could have security implications.